packages/dx-react-scheduler/demos/featured/redux-integration.md

```diff
@@ -0,0 +1,5 @@
+# React Scheduler Redux Integration
```

Comment:
```suggestion
# React Scheduler - Redux Integration
```

Reply:
We often don't use `-` inside the `Demos` section https://devexpress.github.io/devextreme-reactive/react/grid/demos/featured/remote-data/

Reply:
Docs use but demos don't. That's interesting.